docs/book/mlops_stacks/experiment_trackers/mlflow.md

```diff
@@ -0,0 +1,188 @@
+somewhere on-premise or in the cloud, and you would like to connect ZenML to it
+to share the artifacts and metrics logged by your pipelines
+
+You should consider one of the other [Experiment Tracker flavors](./overview.md#experiment-tracker-flavors)
+if you have never worked with MLflow before and would rather use another
+experiment tracking tool that you are more familiar with.
+
+## How do you deploy it?
+
+
+The MLflow Experiment Tracker flavor is provided by the MLflow ZenML
+integration, you need to install it on your local machine to be able to register
+an MLflow Experiment Tracker and add it to your stack:
+
+```shell
+zenml integration install mlflow -y
+```
+
+The MLflow Experiment Tracker can be configured to accommodate the following
+[MLflow deployment scenarios](https://mlflow.org/docs/latest/tracking.html#how-runs-and-artifacts-are-recorded):
+
+* [Scenario 1](https://mlflow.org/docs/latest/tracking.html#scenario-1-mlflow-on-localhost):
+This scenario requires that you use a [local Artifact Store](../artifact_stores/local.md)
+alongside the MLflow Experiment Tracker in your ZenML stack. The local Artifact
+Store comes with limitations regarding what other types of components you can
+use in the same stack. This scenario should only be used to run ZenML locally
+and is not suitable for collaborative and production settings. No parameters
+need to be supplied when configuring the MLflow Experiment Tracker, e.g:
+
+```shell
+# Register the MLflow experiment tracker
+zenml experiment-tracker register mlflow_experiment_tracker --flavor=mlflow
+
+# Register and set a stack with the new experiment tracker
+zenml stack register custom_stack -e mlflow_experiment_tracker ... --set
+```
+
+* [Scenario 5](https://mlflow.org/docs/latest/tracking.html#scenario-5-mlflow-tracking-server-enabled-with-proxied-artifact-storage-access):
+This scenario assumes that you have already deployed an MLflow Tracking Server
+enabled with proxied artifact storage access. There is no restriction regarding
+what other types of components it can be combined with. This option requires
+[authentication related parameters](#authentication-methods) to be configured
+for the MLflow Experiment Tracker.
+
+### Authentication Methods
+
+You need to configure the following credentials for authentication to a remote
+MLflow tracking server:
+
+* `tracking_uri`: The URL pointing to the MLflow tracking server.
+* `tracking_username`: Username for authenticating with the MLflow tracking
+server. 
+* `tracking_password`: Password for authenticating with the MLflow tracking
+server. 
+* `tracking_token`: Token for authenticating with the MLflow tracking server.
+* `tracking_insecure_tls`: Set to skip verifying the MLflow tracking server SSL
+certificate.
+    
+Either `tracking_token` or `tracking_username` and `tracking_password` must be
+specified.
+
+{% tabs %}
+{% tab title="Basic Authentication" %}
```

Comment:
I would have the recommended method as the first tab, ill leave this here also for the rest of the pages